MewsSystems/mews-flutter
# feat: [DX-724] Migrate to sizes tokens

#### Summary

1. Replaced sizes with according tokens.
2. Added todos for components that need more info from the design team.

#### Testing steps

This one is touching almost every component so it would be better to browse through every story and check if components don't look off.

#### Follow-up issues

Fixing stuff from todo's.

#### Check during review

- Verify against Jira issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).
